pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>org.jboss.aerogear.unifiedpush</groupId>
             <artifactId>unifiedpush-push-model</artifactId>
-            <version>1.1.0.Final</version>
+            <version>1.2.0-SNAPSHOT</version>
```

Comment:
LOL - nevermind this comment.

Let's do one of the following:
* get 1.1.3.Final out, soon-ish
* get 1.2.0-alpha.1 out 

and use that - until than, let's not merge it? As it breaks travis?